src/server/pachyderm_test.go

```diff
@@ -105,14 +106,16 @@ func TestSimplePipeline(t *testing.T) {
 	for _, info := range commitInfos {
 		commitRepos = append(commitRepos, info.Commit.Branch.Repo)
 	}
-	require.EqualOneOf(t, commitRepos[:2], client.NewRepo(dataRepo))
-	require.EqualOneOf(t, commitRepos[:2], client.NewSystemRepo(pipeline, pfs.SpecRepoType))
-	require.EqualOneOf(t, commitRepos[2:], client.NewRepo(pipeline))
-	require.EqualOneOf(t, commitRepos[2:], client.NewSystemRepo(pipeline, pfs.MetaRepoType))
+	require.ElementsEqual(t, commitRepos[:2], []*pfs.Repo{client.NewRepo(dataRepo), client.NewSystemRepo(pipeline, pfs.SpecRepoType)})
+	require.ElementsEqual(t, commitRepos[:2], []*pfs.Repo{client.NewRepo(pipeline), client.NewSystemRepo(pipeline, pfs.MetaRepoType)})
```

Comment:
Oh whoops, you can't do ElementsEqual on proto structs because they can't be map keys. I think the previous approach to this was ok?